pkg/cmd/stack/translate.go

```diff
@@ -102,7 +102,7 @@ func translateEnvFile(svc *model.Service, filename string) error {
 	return nil
 }
 
-func translateBuildImages(ctx context.Context, s *model.Stack, noCache bool) error {
+func translateBuildImages(ctx context.Context, s *model.Stack, forceBuild, noCache bool) error {
 	c, _, configNamespace, err := k8Client.GetLocal("")
```

Comment:
should we consider the context of `okteto.yml` here?

Reply:
yes, but this is for stacks and they don't support `context` yet